fixtures/js-stubs/accessRequest.tsx

```diff
@@ -0,0 +1,15 @@
+import {AccessRequest as AccessRequestType} from 'sentry/types';
+
+import {Member} from './member';
+import {Team} from './team';
+
+export function AccessRequest(
+  params: Partial<AccessRequestType> = {}
+): AccessRequestType {
```

Comment:
We're also going to need to update https://github.com/getsentry/sentry/blob/299d07e193818163ef85f0f42c7020744b873014/fixtures/js-stubs/types.tsx#L23

Reply:
i'm happy to see that be separate again, i'm worried it's a big can of worms

Reply:
Ah yes, one sec

Reply:
Type inference for stubs will def need to be fixed/improved, but we need to move them to TS first